tests/client/chain/grpc/test_chain_grpc_exchange_v2_api.py

```
@@ -2647,6 +2647,116 @@ async def test_fetch_l3_spot_orderbook(
+            ]
+        }
+
+        assert market_balances_response == expected_market_balances
+
+    @pytest.mark.asyncio
+    async def test_fetch_denom_min_notional(
+        self,
+        exchange_servicer,
+    ):
+        amount = "1"  # Decimal as a string
+        response = exchange_query_pb.QueryDenomMinNotionalResponse(
```

Pylint (3.3.7)

[error] Module 'pyinjective.proto.injective.exchange.v2.query_pb2' has no 'QueryDenomMinNotionalResponse' member

(E1101)